src/pages/patientView/trialMatch/TrialMatchTable.tsx

```diff
@@ -196,6 +196,12 @@ export default class TrialMatchTable extends React.Component<ITrialMatchProps> {
         );
     }
 
+    public openFeedbackForm(nctId: string, protocolNo: string) {
```

Comment:
this modifies mobx state and so should have @action decorator

Reply:
Added @action